src/application-manager.d.ts

```diff
@@ -65,6 +68,14 @@ export class ApplicationManager extends EventEmitter {
 	// FIXME: Type this properly as it's some mutant state between
 	// the state endpoint and the ApplicationManager internals
 	public getStatus(): Promise<Dictionay<any>>;
+	// The return type is incompleted
+	public getTargetApps(): Promise<
+		Dictionary<{
+			services: Dictionary<Service>;
+			volumes: Dictionary<Volume>;
+			networks: Dictionary<Network>;
+		}>
+	>;
```

Comment:
I'm currently in the process of rewriting this module in typescript, so it would be counter productive to define this fully and the only typescript user is this module.